plugins/statsd/statsd_input.go

```diff
@@ -152,15 +156,34 @@ func parseMessage(message []byte) ([]Stat, error) {
 		lines = [][]byte{message}
 	}
 
-	for _, line := range lines {
+	var num_lines int = len(lines)
+	var bad_lines_ct int = 0
```

Comment:
not a real "go" expert, but doing "length" in a few languages can be a performance issue.  if there is not penalty in Go, no there is no need to keep a separate count.